enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -730,7 +730,7 @@ def remaining_balance_per_user(self, lms_user_id=None):
         Returns the remaining redeemable credit for the user.
         """
         existing_transaction_count = len(self.transactions_for_learner(lms_user_id)['transactions'])
-        return self.per_learner_enrollment_limit - existing_transaction_count
+        return int(self.per_learner_enrollment_limit or 0) - existing_transaction_count
```

Comment:
nit: Is the `int` necessary versus `(self.per_learner_enrollment_limit or 0) - existing_transaction_count`? I believe both would have the same result.

Same question for the other `per_learner_spend_limit` case below.